### PR TITLE
Carousel section

### PR DESCRIPTION
**Pull request by AbbyNyakara**
This is Carousel section. I tried using react slick and another built in tool but they both did not work. They are not customizable. My biggest challenge so far is figuring out how to get the other slides to peak. Im still looking for a way to do it. 
Any suggestion will help.  But the slider is working

### CODE REVIEW

**Comment by AbbyNyakara** [comment]
Hi @gmagnenat , are you able to review the pull request?

File: src/components/Carousel/Carousel.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+.carousel__header {
+   font-size: variables.$fs-9;
+   font-weight: variables.$fw-bold;
+   text-align: center;
+   padding-bottom: 2.5rem;
+}
+
+.carousel__slider {
+    width: 1000px;
```
Comment (gmagnenat) [inline src/components/Carousel/Carousel.scss:16]: This is forcing a strange width, with the display flex.. I'll try to debug later

File: src/components/Carousel/Carousel.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    margin: auto;
+    border-radius: 10px;
+    display: flex;
+  }
+
+.carousel__image {
+    border-radius: 10px;
+    margin-right: 20px;
+    width: 100%;
+    max-height: 180px;
```
Comment (gmagnenat) [inline src/components/Carousel/Carousel.scss:26]: image ratio isn't maintained here. 
object-fit: cover maybe?

File: src/components/Carousel/Carousel.jsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+       <div className="carousel__slider">
+          {
+            imageData.map((slide, index) => {
+              return (
+                <div className = {index === current ? 'slide-active' : slide} key={index}>
+                  {
+                    index === current && (
+                      <img src={slide.imageURL} alt="" className="carousel__image" />
+                    )
+                  }
```
Comment (gmagnenat) [inline src/components/Carousel/Carousel.jsx:36]: It works, but something isn't right and result in a strange loading effect when we switch images. We'll try to debug later.

**Review by gmagnenat — APPROVED** [review]

File: src/components/Carousel/Carousel.jsx (hunk 3 above)
Comment (AbbyNyakara) [reply src/components/Carousel/Carousel.jsx:36]: Okay. I just noticed that

**Pull request merged**